pydotorg/resources.py

```diff
@@ -120,3 +120,4 @@ class Meta:
         authentication = ApiKeyOrGuestAuthentication()
         authorization = StaffAuthorization()
         throttle = CacheThrottle(throttle_at=600) # default is 150 req/hr
+        abstract = True
```

Comment:
Did you get any error without this line? I couldn't find anything related at https://github.com/django-tastypie/django-tastypie/blob/master/docs/release_notes/v0.14.0.rst Or did something changed in Django 2.0?

Reply:
Unfortunately, tp 0.14.1 doesn't have release notes, but is needed for django 2 support.

The need for this is from this commit: https://github.com/django-tastypie/django-tastypie/commit/7dedb7be80868ee5dc72b79226b89bdb66f8a172

A workaround was added here: https://github.com/django-tastypie/django-tastypie/pull/1519

Not ideal, as child classes must be explicitly abstract False, however that's what has been provided

Reply:
Thanks. We are going to remove tastypie soon so we can live with bunch of `abstract = False` additions :)